docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/index.md

```diff
@@ -4,15 +4,17 @@ date: "2022-04-05"
 sidebar_position: 500
 ---
 
-The configuration reference in this page is written for RDB Loader 4.0.0 or higher
+The configuration reference in this page is written for RDB Loader 5.0.0 or higher
```

Comment:
There are two types of previous versions folder:
* [The one for previous config references](https://docs.snowplow.io/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/rdb-loader-configuration-reference/rdb-loader-previous-versions/)
* [The one for main rdb loader section](https://docs.snowplow.io/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/previous-versions/)

I kept the config reference for v4 however we discussed with @stanch and decided that it would better if we don't create v4 section under main previous versions folder because most of the parts are still same. Maybe we can discuss later to decide structure of previous versions folder and what to put there exactly.